docs/crio.8.md

```diff
@@ -105,6 +102,9 @@ set the CPU profile file path
 **--log-format**=""
   Set the format used by logs ('text' (default), or 'json') (default: "text")
 
+**--log-level**=""
+  log CRI-O messages above specified level: debug, warn, error, fatal or panic
```

Comment:
if you change "above" above, make similar changes here please.  Also please capitalize log at the start of the sentence.